### PR TITLE
Merging #2290, #2291-#2308, #2310 to prod

### LINKED ISSUE #2229: Investigate and migrate repo secrets migration from git-crypt to sops.
Sops:
http://github.com/mozilla/sops/

### PR DESCRIPTION
**Pull request by felder**
Converts secrets encryption to sops.

Fixes #2229

### CODE REVIEW

**Comment by felder** [comment]
@yuvipanda I'd like another set of eyes on this please as it's a pretty substantial amount of changes.

Thanks!

File: .circleci/config.yml [hunk 1]
```diff
@@ -269,10 +268,10 @@ jobs:
             mv linux-amd64/helm /usr/local/bin
             helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
 
-
       - run:
           name: Activate credentials for datahub cluster (fall-2019)
           command: |
+            sops -d -i deployments/datahub/secrets/gke-key.json
```
Comment (yuvipanda) [inline .circleci/config.yml:274]: hubploy automatically does this as well, so this isn't needed either.

**Review by yuvipanda — APPROVED** [review]
YAY!

**Comment by yuvipanda** [comment]
Thanks you so much for pushing this through, @felder!

**Pull request merged**